### PR TITLE
Adds defaults to custom fields, closes #2393

### LINKED ISSUE #2393: Add defaults to custom fields
This is the most obvious thing that Personal Text / Blurb can do that custom fields currently can't. Besides being shown without title, which I've already got done in my own code.

### PR DESCRIPTION
**Pull request by Vekseid**

### CODE REVIEW

**Comment by Vekseid** [comment]
I am curious how my test forum still ran with that.

File: sources/Load.php [hunk 1]
```diff
@@ -1020,7 +1020,7 @@ function loadMemberContext($user, $display_custom_fields = false)
 	// Set things up to be used before hand.
 	$profile['signature'] = str_replace(array("\n", "\r"), array('<br />', ''), $profile['signature']);
 	$profile['signature'] = $parsers->parseSignature($profile['signature'], true);
-	$profile['is_online'] = (!empty($profile['show_online']) || allowedTo('moderate_forum')) && $profile['is_online'] > 0;
+	$profile['is_online'] = !empty($profile['show_online']) || (allowedTo('moderate_forum') && $profile['is_online'] > 0);
```
Comment (emanuele45) [inline sources/Load.php]: Is this related to #2413 ?

Comment (Vekseid) [inline sources/Load.php]: Actually that's the wrong grouping. Ack.

And not exactly, I saw the useless parens while trying to figure out what was going on with it.
